### PR TITLE
feat: Update builder homepage screen

### LINKED ISSUE #2345: Builder Homepage
Today [builder.decentraland.org](https://builder.decentraland.org/) is the equivalent of the Scene Builder subsection which might not be very relevant depending on the use case.

This can be especially confusing for first time users who don’t know yet all the tools they can find in this Builder section.

That is why we are proposing a Homepage for the Builder section that **introduces each subsection and also offers quick links to related documentations**.

For returning users, we propose saving the last subsection that was used so they don’t need to go through this homepage every time.

Tasks:
- Rename `HomePage` to `ScenesPage` and change route to `/scenes`
- Create `OverviewPage` as described in this issue and assign it to the route `/`
- Reorder the tabs in `Navigation`

### PR DESCRIPTION
**Pull request by cyaiox**
This PR renames the current `homepage screen` to `scenes screen` and creates a new homepage overview.

image

Closes #2345

### CODE REVIEW

File: src/components/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -1,203 +1,108 @@
...
-  renderImportButton = () => {
-    return (
-      <Button basic className="import-scene" onClick={this.handleOpenImportModal}>
-        <BuilderIcon name="import" />
-      </Button>
-    )
-  }
+export const HomePage: React.FC<Props> = props => {
+  const { isLoggingIn, isLoggedIn, onNavigate } = props
+  const lastVisitedSection = localStorage.getItem(LOCALSTORAGE_LAST_VISITED_SECTION_KEY)
```
[the remark this answers is not in the document]
Comment (cyaiox) [reply src/components/HomePage/HomePage.tsx:23]: great idea! I'll be adding it soon 🙌

File: src/components/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -1,203 +1,108 @@
...
-                )
-              }}
-            />
-          </div>
-        </div>
-      )
+  const getLearnMoreLink = (card: string): string => {
+    switch (card) {
+      case NavigationTab.COLLECTIONS:
+        return 'https://docs.decentraland.org/creator/builder/builder-101/#collections'
```
[the remark this answers is not in the document]
Comment (cyaiox) [reply src/components/HomePage/HomePage.tsx]: Hey @juanmahidalgo -- thanks for the feedback! I've just updated the associated links

**Review by fzavalia — CHANGES_REQUESTED** [review]
Going to collections via overview and clicking the back button on the browser will not take me back to overview. It keeps looping on the collections tab. Can we fix that?

**Comment by fzavalia** [comment]
> Going to collections via overview and clicking the back button on the browser will not take me back to overview. It keeps looping on the collections tab. Can we fix that?

It happens for all tabs as well

**Comment by fzavalia** [comment]
> Going to collections via overview and clicking the back button on the browser will not take me back to overview. It keeps looping on the collections tab. Can we fix that?

I think it happens due to the useEffect in the HomePage which redirects back

**Review by fzavalia — APPROVED** [review]

**Pull request merged**